pages/clustering/high-availability.mdx

```diff
@@ -138,8 +139,9 @@ export MEMGRAPH_EXPERIMENTAL_ENABLED=high-availability
 export MEMGRAPH_COORDINATOR_PORT=10111
 export MEMGRAPH_COORDINATOR_ID=1
 export MEMGRAPH_BOLT_PORT=7692
+export MEMGRAPH_NURAFT_LOG_FILE="<path-to-log-file>"
 ```
-When using either of these flags, configurations flags for `--bolt-port`, `--coordinator-port`, `--coordinator-id` and `--experimental-enabled` will be ignored.
+When using either of these environment variables, flags for `--bolt-port`, `--coordinator-port`, `--coordinator-id` and `--experimental-enabled` will be ignored.
```

Comment:
I've updated the statement to use "any" instead of "either". -> https://github.com/memgraph/documentation/pull/804/commits/0002026276044e905db75dbe5363dbc073f6b339